PetScanList/text_bot.py

```
@@ -96,38 +100,44 @@ def format_list_as_text(p_list, other_params):
-        return {}, "no_template"
+        return {}, NO_TEMPLATE_MESSAGE
 
     p_list, other_params = make_petscan_list(template)
-
-    if not p_list or len(p_list) == 1 and p_list[0] == "":
-        return {}, "no_result_petscan"
+    
+    if not p_list or len(p_list) == 0:
+        return {}, NO_RESULT_MESSAGE
+    
+    if type(p_list) == list and  p_list[0] == "":
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)